### PR TITLE
CU-86b0w6z6c_Game-notification

### PR DESCRIPTION
**Pull request by Victorportus**
Clicking on the "Show New Level Modal" will show the associated modal, to close click outside of the modal

### CODE REVIEW

**Comment by zeckkimon** [comment]
Task linked: [CU-86b0w6z6c Game notification](https://app.clickup.com/t/86b0w6z6c)

File: src/components/modals/NewLevelModal.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+const NewLevelModal: React.FC<NewLevelModalProps> = ({
+  isVisible,
+  onClose,
+}) => {
+  return (
+    <Modal isVisible={isVisible} onBackdropPress={onClose}>
+      <View style={styles.modalContent}>
+        <Text style={styles.title}>You have reached Level 3</Text>
```
Comment (whamester) [inline src/components/modals/NewLevelModal.tsx]: Text should come from the new custom Text component we have. the styles should match the figma design already, if they don't match we can review it.

Comment (Victorportus) [reply src/components/modals/NewLevelModal.tsx]: Implemented text with levels

**Review by whamester — APPROVED** [review]

**Review by zeckkimon — APPROVED** [review]

**Review by ReinhardtBotha — APPROVED** [review]
Good job!

**Pull request merged**